index.html

```diff
@@ -7,12 +7,143 @@
     <link rel="preconnect" href="https://fonts.googleapis.com">
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
     <link href="https://fonts.googleapis.com/css2?family=Lato&display=swap" rel="stylesheet">
-    <link href="index.css" rel="stylesheet";
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://unicons.iconscout.com/release/v4.0.8/css/line.css">
+    <link href="index.css" rel="stylesheet">
     <title>Junior Achievers Kenya</title>
 
 </head>
 
 <body>
+
+    <header>
+                   
+        
+        <nav>
+            
+            <div class="top">
+                <a href="#"><i class="uil uil-facebook-f"></i></a>
+                <a href="#"><i class="uil uil-twitter"></i></a>
+                <a href="#">English</a>
+                <a href="#">My Page</a>
+                <a href="#">Logout</a>
+    
+                </div>
+                
+            <div class="bottom">
+            <a href="index.html" class="logo">JA KENYA</a>
+
+            <ul class="nav-menu">
+                <li class="nav-item"><a href="about.html" class="nav-link">About</a></li>
+                <li class="nav-item"><a href="#" class="nav-link">Program</a></li>
+                <li class="nav-item"><a href="#" class="nav-link">Join</a></li>
+                <li class="nav-item"><a href="#" class="nav-link">Sponsor</a></li>
+                <li class="nav-item"><a href="#" class="nav-link">News</a></li>
+                <li><a href="#" class="nav-link">CC Campaign</a></li>
+                
+            </ul>
+
+           
+
+            <div class="hamburger">
+                <span class="bar"></span>
+                <span class="bar"></span>
+                <span class="bar"></span>
+            </div>
+
+        </div>
+        </nav>
+        
+    </header>
+
+    <section class="hero">
+        <div class="hero-wrapper">
+        <p class="hello">"Hello Sharing world</p>
+        <h1>CREATIVE COMMONS <br /> GLOBAL SUMMIT <br />2015</h1>
+        <p class="description">A joyful celebration believing in the value of openness and sharing, creating a positive change with people from over 80 countries is taking place in October, in Korea.</p>
+        <h2 class="date">2015.10.15(THU) ~ 16(FRI)</h2>
+        <p class="location">@ National Museum of Karea, Art Center Nabi and more
+        </p>
+    </div>
+    </section>
+
+    <section class="program">
+        <div class="program-wrapper">
+
+            <h2 class="program-title">Main Program</h2>
```

Comment:
- [ ] KIndly check out the way the underline for each section title are designed and the color assigned to them. Yours look different

design guideline 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/235224678-5b263a94-7bcd-470a-9a4b-4e6318c96db1.png)

your design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/235224827-6942e0b6-899a-4ac1-9140-a30741ccc1f2.png)